doc/howtos/authorized-keys.rst

```diff
@@ -0,0 +1,104 @@
+.. _authorized-keys:
+
+Adding Authorized Keys for Remote ssh Access
+############################################
+
+In Ostro |trade| OS **development** images only, root automatically gets logged in on a local 
+console or serial port connection. By default though, Ostro OS prevents remote logins to your 
+target device.  You can enable remote access as root via ``ssh`` in your development image by installing 
+your personal public key in the ``~root/.ssh/authorized_keys`` file.  This tech note explains how.
+
+You’ll first need to have a private/public key pair on your workstation to enable ``ssh`` access to the target device.  You can use
+an existing host ssh key pair, found in ``$HOME/.ssh``. 
+A private/public key pair will be stored in two files for example, ``.ssh/id_rsa`` and ``.ssh/id_rsa.pub``.
+
+If you don't see private/public key files, you'll need to generate them.
+
+Generating a Private/Public Key Pair
+====================================
+
+On Linux systems (or Windows systems with `Git Bash`_ installed) use the ``ssh-keygen`` utility to generate
+an ssh key pair by following its instructions ::
+
+   $ ssh-keygen -t rsa -b 4096 -C "your_email@example.com"
+
+   # Creates a new ssh key using the provided email as a label
+   Generating public/private rsa key pair.
+   Enter a file in which to save the key ($HOME/.ssh/id_rsa): [Press enter]
+   # Press enter to accept the default file names or provide a name for development use, for example
+
+   Enter passphrase (empty for no passphrase): [Type a passphrase]
+   # If you're only using this key for local development work, you could skip a passphrase and press Enter
+   Enter same passphrase again: [Type passphrase again]
+
+At this point the tool will generate an ssh key pair and store the public and private keys in separate files in the ``$HOME/.ssh/`` folder.
+
+On Windows systems, you can also use **PuTTYgen**.  If needed, download PuTTYgen from the `PuTTY download page`_ (you may already have
+it installed on your Windows system).  Launch the program and click the **Generate** button.  (If you're only using this key for 
+local development work, you could skip a passphrase.) Save the key-pair files by clicking the **Save public key** and **Save private key** buttons.  
+
```

Comment:
saved public key by puttygen would not be directly usable as content  into authorized_keys on device.
Line for authorized_keys in puttygen should be copy&pasted from dialog windows to some file.